ts/packages/ts/src/commands/commands.test.ts

```diff
@@ -221,5 +243,147 @@ describe(`commands`, () => {
+        expect.assertions(1);
+        await expect(() =>
+          new EvalCommand({
+            store: store1,
+            set: set1,
+            schema: new ConfigSchema('lazy', {
+              K1: {
+                type: 'String',
+                lazy: true,
+                required: true,
+              },
+            }),
+          }).run(),
+        ).rejects.toBeInstanceOf(ConfigError);
+      });
+      test('run EvalCommand without override for lazy config when `cfgu.required = true`', async () => {
+        expect.assertions(1);
+        await expect(() =>
+          new EvalCommand({
+            store: store1,
+            set: set1,
+            schema: new ConfigSchema('lazy', {
+              K1: {
+                type: 'String',
+                lazy: true,
+                required: true,
+              },
+              K2: {
+                type: 'String',
+              },
+            }),
+            configs: {
+              K2: '2',
+            },
+          }).run(),
+        ).rejects.toBeInstanceOf(ConfigError);
+      });
+      test('EvalCommand with override for lazy config when `cfgu.required = false`', async () => {
+        expect.assertions(1);
+        await expect(() =>
+          new EvalCommand({
+            store: store1,
+            set: set1,
+            schema: new ConfigSchema('lazy', {
+              K1: {
+                type: 'String',
+                lazy: true,
+                required: true,
+              },
+              K2: {
+                type: 'String',
+              },
+            }),
+            configs: {
+              K2: '2',
+            },
+          }).run(),
+        ).rejects.toBeInstanceOf(ConfigError);
+      });
+      test('run EvalCommand with override for lazy config when `cfgu.required = false`', async () => {
+        const result = await new EvalCommand({
+          store: store1,
+          set: set1,
+          schema: new ConfigSchema('lazy', {
+            K1: {
+              type: 'String',
+              lazy: true,
+              required: false,
+            },
+          }),
+          configs: {
+            K1: '1',
+          },
+        }).run();
+        expect(result).toMatchObject({ K1: { result: { value: '1' } } });
+      });
+      test('run EvalCommand with override for lazy config when `cfgu.required = true`', async () => {
+        const result = await new EvalCommand({
+          store: store1,
+          set: set1,
+          schema: new ConfigSchema('lazy', {
+            K1: {
+              type: 'String',
+              lazy: true,
+              required: true,
+            },
+          }),
+          configs: {
+            K1: '1',
+          },
+        }).run();
+        expect(result).toMatchObject({ K1: { result: { value: '1' } } });
+      });
+      test('run EvalCommand without override for lazy config when `cfgu.required = false`', async () => {
```

Comment:
the `required` prop is not defined, we need to clarify the test names and have them align with what's going on inside the test.

Reply:
all tests names have been refactored. please review.